mailpoet/lib/Automation/Integrations/WooCommerce/Triggers/Orders/OrderStatusChangedTrigger.php

```diff
@@ -19,10 +19,10 @@
 class OrderStatusChangedTrigger implements Trigger {
 
   /** @var Functions */
-  private $wp;
+  protected $wp;
```

Comment:
Is there a reason we sometimes use `WPFunctions` and sometimes use `WordPress` in the automations codebase?

Reply:
Good catch! Thank you. Replaced all `Functions` with the `Engine\WordPress` in 022bf27a1